packages/amplify-category-function/src/provider-utils/awscloudformation/utils/layerParams.ts

```diff
@@ -155,7 +165,7 @@ export const getLayerMetadataFactory = (projectBackendDirPath: string): LayerMet
       return undefined;
     }
     const parametersFilePath = path.join(resourceDirPath, layerParametersFileName);
-    const obj = JSON.parse(fs.readFileSync(parametersFilePath, 'utf8')).parameters;
+    const obj = JSON.parse(fs.readFileSync(parametersFilePath, 'utf8'));
```

Comment:
would be nice to use context.amplify.readJsonFile()

We could pass in context instead of projectBackendDirPath and compute projectBackendDirPath in this function.

If that's going to be difficult to change, then just leave it